<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ChatRoomName">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ChatRoom name</string>
    </property>
   </widget>
   <widget class="QLabel" name="username">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Join">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>340</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Join</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TextChatRoomName">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TextUsername">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="UserMessage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SendMessage">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>430</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>441</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveSettings">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>280</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>SaveSettings</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
